content/en/docs/ops/configuration/security/migrate-alpha-policy/index.md

```diff
@@ -0,0 +1,338 @@
+upgrade to Istio 1.5 as a transitive version because it supports both `v1alpha1` and `v1beta1` security policy. You complete
+the security policy migration in Istio 1.5, remove the `v1alpha1` security policy and then continue the upgrade to later Istio versions.
+
+Istio 1.5 is the only version that supports both `v1alpha1` and `v1beta1` security policy. For a given workload, the
+`v1beta1` version will take precedence over the `v1alpha1` version.
+
+Alternatively, if you want to do a skip-upgrade directly from Istio 1.4 to Istio 1.6 or later, you should use the
+[canary upgrade](/docs/setup/upgrade/canary/) to install the new Istio version as a separate control plane, gradually
+migrate your workloads to the new control plane and complete the security policy migration at the same time.
+
+Note that skip-upgrade is not supported by Istio and there might be other issues in this process. Istio 1.6 does not support
+the `v1alpha1` security policy, and if you do not migrate them to `v1beta1` version, you are essentially deleting all your
+`v1alpha1` security policy.
+
+In either case, it is recommended migrating using namespace granularity: for each namespace, find all
+`v1alpha1` policies that have an effect on the workloads in the namespace and migrate all the `v1alpha1` policies at the same time.
+This allows a safer migration as you can apply the `v1beta1` policies one namespace at a time, make sure everything is working
+as expected and then move forward to the next namespace.
+
+## Major Differences
+
+Before starting the migration, read through the `v1beta1` [authentication](/docs/concepts/security/#authentication)
+and [authorization](/docs/concepts/security/#authorization) documentation to understand the `v1beta1` policy.
+
+You should examine all of your existing `v1alpha1` security policies, find out what fields are used and which policies
+need migration, compare the findings with the major differences listed below and confirm there are no blocking issues
+(e.g. using some alpha feature that is no longer supported in beta):
+
+| Major Differences | `v1alpha1` | `v1beta1` |
+|---------|------------------------|--------------------------------|
+| API stability | not backward compatible | backward compatible |
+| mTLS | `MeshPolicy` and `Policy` | `PeerAuthentication` |
+| JWT | `MeshPolicy` and `Policy` | `RequestAuthentication` |
+| Authorization | `ClusterRbacConfig`, `ServiceRole` and `ServiceRoleBinding` | `AuthorizationPolicy` |
+| Policy target | service name based | workload selector based |
+| Port number | service ports | workload ports |
+
+Although `RequestAuthentication` in `v1beta1` security policy is similar to `v1alpha1` JWT policy, there is a notable
+semantics change. The `v1alpha1` JWT policy needs to be migrated to two `v1beta1` resources: `RequestAuthentication` and `AuthorizationPolicy`.
+
+This will change the JWT deny message due to the use of `AuthorizationPolicy`. In alpha version, the HTTP code 401 is returned
```

Comment:
```suggestion
This will change the JWT deny message due to the use of `AuthorizationPolicy`. In the alpha version, the HTTP code 401 is returned
```
```suggestion
This will change the JWT deny message due to the use of `AuthorizationPolicy`. In alpha version, the HTTP code 401 is returned
```